src/plugins/condition/components/Condition.vue

```diff
@@ -21,160 +21,167 @@
 *****************************************************************************/
 
 <template>
-<div v-if="isEditing"
-     class="c-condition c-condition--edit js-condition-drag-wrapper"
+<div class="c-condition-h"
+     :class="{ 'is-drag-target': draggingOver }"
+     @dragover.prevent
+     @drop.prevent="dropCondition($event, conditionIndex)"
+     @dragenter.prevent="dragEnter($event, conditionIndex)"
```

Comment:
We dont need to prevent on dragenter and dragleave

Reply:
done